src/main/java/Week2/src/main/Ui.java

```diff
@@ -43,4 +34,18 @@ public void hello() {
         System.out.println("Hello! I'm Duke\nWhat can I do for you?");
         Duke.lining();
     }
+
+    public void bye() {
+        Duke.lining();
+        System.out.println("Bye. Hope to see you again soon!");
+        Duke.lining();
+    }
+
+    public void showEmptyError() {
```

Comment:
I like the naming of the methods that makes it really clear as to what the function does.